packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -145,6 +159,46 @@ export default {
     .card-header:hover .seamless-button {
         opacity: 1;
     }
+
+    /*
+     * Gives the faded content effect for preview.
+     * Ensure that height has the same value as collapsedPanelHeight in PanelBase.js.
+     */
+    .card-preview-collapsed::after {
+        content: "";
+        position: absolute;
+        width: 100%;
+        bottom: 0;
+        height: 125px;
+        background-image: linear-gradient(180deg, transparent, white 90%);
+    }
+
+    .preview-read-more {
+        z-index: 1;
+        opacity: 0.2;
+    }
+
+    /* Targets the before pseudoelement of glyphicon-chevron-down. */
+    .preview-read-more::before {
+        position: absolute;
+        width: 100%;
+        text-align: center;
+        bottom: 10px;
+    }
+
+    .preview-read-more:hover {
+        cursor: pointer;
+        opacity: 0.4;
+    }
+
+    .preview-read-more-fade-enter-active, .preview-read-more-fade-leave-active {
```

Comment:
could move `transition: opacity 0.5s;` into the main `preview-read-more` class

Reply:
Ah, thanks for the suggestion. It makes the code more concise. Will make the changes.